specs/0116-LLM-assisted-translations.md

```diff
@@ -0,0 +1,51 @@
+- Feature Name: LLM-Assisted Translations in Pontoon
+- Created: 2024-01-12
+- Associated Issue: #3064
+
+# Summary
+
+Introduce LLM-assisted translations in Pontoon to provide alternative, formal, and informal translation options, enhancing translation quality.
+
+# Motivation
+
+Modern Large Language Models (LLMs) — like OpenAI’s GPT-4 — have reached a quality level for localization tasks that is on par with dedicated machine translation engines. This feature is primarily focused on introducing a tool that aims to improve translation quality in Mozilla's projects. By incorporating LLM-assisted translations, Mozilla not only remains competitive with other Translation Management Systems (TMS) providers but also embraces new technology to enhance the overall translation process. This feature also lays the groundwork for future integrations that could further enhance efficiency and broaden the scope of translation improvements within Mozilla's localization efforts.
+
+# Feature explanation
+
+In the Machinery tab of the Pontoon translate view, users will encounter a new drop-down option for each machine translation suggestion for a given string, offering four choices: 
+1) `"SHOW ORIGINAL"` - Restore the translation that was initially recommended  by machine translation 
```

Comment:
```suggestion
1) `SHOW ORIGINAL` - Restore the translation that was initially recommended by machine translation 
```

Since you're using backticks, I don't think you should keep quotes (they're not actually used in the labels).

Reply:
Maybe also note that this option is only visible if one of the other options has already been applied. For that reason, I'd put it last.

Reply:
Makes sense, I added a note on how the `SHOW ORIGINAL` option will only be seen once a new translation is generated.